docs/source/daml/intro/8_Exceptions.rst

```diff
@@ -11,30 +11,35 @@ from an error and continue the transaction instead of aborting it.
 
 One option for doing that is to represent errors explicitly via
 ``Either`` or ``Option`` as shown in :doc:`3_Data`. This approach has
-the advantage that it is very explicit about which operations can fail
-and which cannot. However, it also has some downsides. First, it can
-be invasive for operations where aborting the transaction is often the
-desired behavior, e.g., changing division to return ``Either`` or an
-``Option`` to handle division by zero would be a very invasive change
-and many callsites might not want to handle the error case explicitly.
-Second, and more importantly, this approach does not allow rolling
-back ledger actions that have happened up to this point: If a contract
-got created before we hit the error, there is no way to undo this.
-
-Exceptions provide a way to handle certain types of errors in a
-non-invasive way and to roll back parts of the transaction that
-lead up to this error. All of that still happens within the same
-transaction and is thereby atomic contrary to handling the error
-outside of Daml.
+the advantage that it is very explicit about which operations are
+allowed to fail without aborting the entire transaction. However, it
+also has two major downsides. First, it can be invasive for operations
+where aborting the transaction is often the desired behavior, e.g.,
+changing division to return ``Either`` or an ``Option`` to handle
+division by zero would be a very invasive change and many callsites
+might not want to handle the error case explicitly. Second, and more
+importantly, this approach does not allow rolling back ledger actions
+that have happened before the point where failure is detected; if a
+contract got created before we hit the error, there is no way to undo
+that except for aborting the entire transaction (which is what we were
+trying to avoid in the first place).
+
+By contrast, exceptions provide a way to handle certain types of
+errors in such a way that, on the one hand, most of the code that is
+allowed to fail can be written just like normal code, and, on the
+other hand, the programmer can clearly delimit which part of the
+current transaction should be rolled back on failure. All of that
+still happens within the same transaction and is thereby atomic
+contrary to handling the error outside of Daml.
 
 .. hint::
 
-  Remember that you can load all the code for this section into a folder called ``8_Exceptions`` by running ``daml new 8Exceptions --template daml-intro-8``
+  Remember that you can load all the code for this section into a folder called ``8_Exceptions`` by running ``daml new 8_Exceptions --template daml-intro-8``
 
 Our example for the use of exceptions will be a simple shop
 template. Users can order items by calling a choice and transfer money
 (in the form of an Iou issued by their bank) from their account to the
-owner in return.
+owner's in return.
```

Comment:
This is not quite right. We transfer it to the owner via an `Iou` but we do not transfer it directly to their account.